src/components/program/ProgramCourses.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useContext } from 'react';
+
+import moment from 'moment';
+import { Collapsible } from '@edx/paragon';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faAngleDown, faAngleUp, faCalendarAlt } from '@fortawesome/free-solid-svg-icons';
+
+import { ProgramContext } from './ProgramContextProvider';
+
+const DATE_FORMAT = 'MMM D, YYYY';
+
+const getCourseRun = course => (
+  course.courseRuns?.sort(
+    (a, b) => (moment(a.start) < moment(b.start) ? 1 : -1),
```

Comment:
what is the logic to get a desired courserun from list of courseruns?

Reply:
This function is used to get the latest course run, I have also added a comment.